service/internal/credentials/vault.go

```diff
@@ -287,16 +289,31 @@ func (v VaultProvider) GetTarget(projectName, targetName string) (responses.Targ
 		return responses.TargetProperties{}, ErrTargetNotFound
 	}
 
+	// These should always exist.
 	roleArn := sec.Data["role_arns"].([]interface{})[0].(string)
-	policyArns := sec.Data["policy_arns"].([]interface{})
 	credentialType := sec.Data["credential_type"].(string)
 
-	var policies []string
-	for _, v := range policyArns {
-		policies = append(policies, v.(string))
+	// Optional.
+	policies := []string{}
+	val, ok := sec.Data["policy_arns"]
+	if ok {
```

Comment:
nit: any reason this one is 2 separate lines vs the one below?

Reply:
ah, yep. was a refactor. will update